go.mod

```diff
@@ -22,23 +22,24 @@ require (
 	go.uber.org/multierr v1.7.0
 	google.golang.org/grpc v1.40.0
 	google.golang.org/protobuf v1.27.1
-	k8s.io/api v0.0.0
-	k8s.io/apiextensions-apiserver v0.0.0
-	k8s.io/apimachinery v0.23.5
+	k8s.io/api v0.24.1
+	k8s.io/apiextensions-apiserver v0.24.1
+	k8s.io/apimachinery v0.24.1
```

Comment:
Even though this is a separate go module, we should match 0.23.5 that kcp core is using, right?

Reply:
This was a consequence of v0.24.1 in pkg/apis. Fixed.